ocpp/v201/datatypes.py

```diff
@@ -555,7 +555,7 @@ class UnitOfMeasureType:
     UnitOfMeasureType is used by: SampledValueType
     """
 
-    unit: Optional[str] = None
+    unit: Optional[enums.StandardizedUnitsOfMeasureType] = None
```

Comment:
I'm a little concerned about this one, because UnitOfMeasure is extendable.  Maybe its okay, but we'll probably have to expand it one day.